locales/fr-FR/about.html

```diff
@@ -1,19 +1,19 @@
 <h2>A propos de ce projet</h2>
 <div class="p">
-    <h3>What is PoGo-UWP?</h3>
-    <span>PoGo-UWP is an UWP (Universal Windows Platform) client for Niantic's Pokemon™ Go Android/iOS game. Being a client, this means that it gives you the ability to play in the same game-world as your friends that are playing with an Android or iOS device.</span>
+    <h3>Qu'est-ce que PoGo-UWP?</h3>
+    <span>PoGo-UWP est une Plateforme Windows Universelle (UWP) pour la version Android/iOS de Niantic's Pokemon™ Go. En tant que client, elle donne la possibilité de jouer dans le même environnement que vos amis qui sont sous Android ou iOS..</span>
```

Comment:
"PoGo-UWP est une application pour la plateforme universelle Windows (UWP), portée depuis la version Android/iOS..."
j'hésite sur la tournure, tu en penses quoi?

EN: I have a doubt on how to say it in FR asking for opinion ;)